python/ray/serve/config.py

```diff
@@ -520,6 +520,19 @@ def metrics_interval_s_deprecation_warning(cls, v: PositiveFloat) -> PositiveFlo
             )
         return v
 
+    @validator("look_back_period_s", always=True)
+    def look_back_period_s_valid(cls, v: PositiveFloat, values):
+        # Get metrics_interval_s from values, or use default if not set
+        metrics_interval_s = values.get(
+            "metrics_interval_s", DEFAULT_METRICS_INTERVAL_S
+        )
+        if v <= metrics_interval_s:
+            raise ValueError(
+                f"look_back_period_s ({v}) must be greater than "
+                f"metrics_interval_s ({metrics_interval_s})!"
+            )
```

Comment:
i am thinking failing loudly may be too aggressive. How about we raise warning for now, then in 2 releases start raising exception.